libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxNavigationTest.kt

```diff
@@ -289,10 +289,10 @@ class MapboxNavigationTest {
     }
 
     @Test
-    fun onDestroySetsRouteToNullInTripSession() {
+    fun onDestroySetsRoutesToEmptyInTripSession() {
         mapboxNavigation.onDestroy()
 
-        verify(exactly = 1) { tripSession.route = null }
+        verify(exactly = 1) { directionsSession.routes = emptyList() }
```

Comment:
directionsSession is supposed to update tripsSession.route

so our real source of truth is the directionsSession

it's a bit more complicated to wire up tests to have this assumption built in